### PR TITLE
javascript1-week3/nada

### PR DESCRIPTION
**Pull request by NadaJrad**

### CODE REVIEW

File: 3_javascript/javascript1/week3/script.js [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+//Series duration of my life
+const seriesDurations = [
+    {
+      title: "Game of thrones",
+      days: 3,
+      hours: 1,
+      minutes: 0,
+    },
+    {
+      title: "friends",
```
Comment (shpomp) [inline 3_javascript/javascript1/week3/script.js:53]: Not a coding thing, but a nitpick: titles should be capitalised 🙏🏽

File: 3_javascript/javascript1/week3/script.js [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+}
+
+logOutSeriesText();
+
+//NOnoN0nOYes
+const notes = [];
+
+function saveNote(content, id) {
+  // write some code here
+  const note = { content, id };
```
Comment (shpomp) [inline 3_javascript/javascript1/week3/script.js:88]: Nice formulation of the object! ⭐

File: 3_javascript/javascript1/week3/script.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+            return note
+        }
+    } 
+    console.log(`Error: No note found with the id ${id}`);
+    return
+}
+  
+const undefinedNote = getNote();
+const firstNote = getNote(1);
+const fnotFoundNote = getNote(3);
```
Comment (shpomp) [inline 3_javascript/javascript1/week3/script.js:116]: typo? `fnot` instead of `not` 🙏🏽

**Review by shpomp — APPROVED** [review]
Great job, Nada! 
I did not find anything much to improve, only some minor nitpicks, not even so much code-related. 
Your naming and structuring is very good - clean and clear, which for some takes a long time to learn, so keep it up that way! 💪🏽 
Just a very nice job 👏🏽

**Comment by NadaJrad** [comment]
> Great job, Nada! I did not find anything much to improve, only some minor nitpicks, not even so much code-related. Your naming and structuring is very good - clean and clear, which for some takes a long time to learn, so keep it up that way! 💪🏽 Just a very nice job 👏🏽

Thanks a lot karo for your feedback and for your awesome support ,  
I will address your comments and close the PR.

**Pull request closed without merging**